src/msg/async/rdma/RDMAConnectedSocketImpl.cc

```diff
@@ -603,7 +603,7 @@ void RDMAConnectedSocketImpl::notify()
   int ret;
 
   ret = write(notify_fd, &i, sizeof(i));
-  assert(ret = sizeof(i));
+  assert(ret == sizeof(i));
```

Comment:
@joscollin Yes we can replace whole function as something:
void RDMAConnectedSocketImpl::notify()
{
  int i = 1;
  assert(sizeof(notify_fd) == write(notify_fd, &i, sizeof(i)));
}

Changed type of i, because notify_fd is 
msg/async/rdma/RDMAStack.h
int notify_fd = -1;